### PR TITLE
copying wp-content to wp temp directory, forcing overwrite in temp di…

### PR DESCRIPTION
**Pull request by kimpenguin**
…rectory if file or folder exists, and updating core version in composer

### CODE REVIEW

File: bin/git-push.sh [hunk 1]
```diff
@@ -93,6 +94,7 @@ function env_branch {
 }
 
 function add_remote {
+  echo "add_remote"
```
Comment (devowhippit) [inline bin/git-push.sh]: Is this echo needed?

File: bin/deploy.sh [hunk 2]
```diff
@@ -106,26 +107,26 @@ if [[ $selection == 0 ]]; then
...
-		source $SCRIPT_PATH/rsync-config.sh $userProj
-	elif [[ $selection2 == 1 ]]; then
-		source $SCRIPT_PATH/rsync-uploads.sh $userProj
-	fi
+  echo "[0] Upload config.yml"
+  echo "[1] Download uploads"
+  printf "Selection: "
+  read selection2
+  if [[ $selection2 == 0 ]]; then
+    source $SCRIPT_PATH/rsync-config.sh $userProj
```
Comment (devowhippit) [inline bin/deploy.sh:126]: I realize in this script we are using an .env file that lists all of the environments. I wonder if it's possible to use the list of git remote names, as the git push script relies on.

**Review by devowhippit — APPROVED** [review]
I think the echo in `git-push.sh` was left over but looks great!

File: bin/git-push.sh (hunk 1 above)
Comment (kimpenguin) [reply bin/git-push.sh]: What good eyes you have! Was testing and left that little buddy in there. Will remove.

File: bin/deploy.sh (hunk 2 above)
Comment (kimpenguin) [reply bin/deploy.sh:126]: Oh totally. We can prompt the user the same we way we do for the deploy selection. Haven't gotten to that task yet, but will likely do that before the next pre-production deployment. Thanks!

**Pull request merged**